lib/__init__.py

```diff
@@ -0,0 +1,19 @@
+__version__ = '1.1.0'
```

Comment:
My project was using the lib items directly, and not calling the top-level script, so I _believe_ we need to have init.py for the module to be usable.  I can test it out and verify.